operator/internal/route/translation.go

```diff
@@ -0,0 +1,58 @@
+package route
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	"github.com/stackrox/rox/operator/internal/values/translation"
+	"github.com/stackrox/rox/pkg/k8sutil"
+	"helm.sh/helm/v3/pkg/chartutil"
+	corev1 "k8s.io/api/core/v1"
+	ctrlClient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	tlsSecretCAKey = "ca.pem"
+	tlsSecretName  = "central-tls"
```

Comment:
Please instead extract the string from [this line](https://github.com/stackrox/stackrox/blob/42c09dfa2f7f59f3e90f0785b79a09dca7e6c3e8/operator/internal/central/extensions/reconcile_tls.go#L67) into an exported const somewhere (`operator/internal/common/tls.go`?) and use it below.